include/tvm/ir/op.h

```diff
@@ -270,14 +270,17 @@ class OpRegEntry {
    *  an higher priority level attribute
    *  will replace lower priority level attribute.
    *  Must be bigger than 0.
+   * \param can_override Whether to explicitly allow
+   *  overriding the attribute, any non-zero value
+   *  implies allowance and 0 means disallowance.
    *
    *  Cannot set with same plevel twice in the code.
    *
    * \tparam ValueType The type of the value to be set.
    */
   template <typename ValueType>
   inline OpRegEntry& set_attr(const std::string& attr_name,  // NOLINT(*)
-                              const ValueType& value, int plevel = 10);
+                              const ValueType& value, int plevel = 10, int can_override = 0);
```

Comment:
Wju do we need to introduce can_override when we already have plevel to specify overriding rule.

Previously we check the plevel, if the plevel have higher priority level then we can override the attribute

Reply:
Yeah that make sense. Thanks for the explanation.